clang/test/SemaCXX/PR41441.cpp

```diff
@@ -1,5 +1,5 @@
-// RUN: %clang -S -fno-discard-value-names -emit-llvm -o - %s | FileCheck %s
-// Issue no: 41441
+// RUN: %clang --target=x86_64-pc-linux -S -fno-discard-value-names -emit-llvm -o - %s | FileCheck %s
+
 #include <new>
```

Comment:
This #include is also possible to cause breakages, you should just put hte important declarations in text.

Reply:
It does:

https://lab.llvm.org/buildbot/#/builders/280/builds/1989
```
/Users/buildbot/buildbot-root/aarch64-darwin/llvm-project/clang/test/SemaCXX/PR41441.cpp:3:10: fatal error: 'new' file not found
    3 | #include <new>
      |          ^~~~~
1 error generated.
```

Reply:
Blarg, the buildbots i checked on were mangling differences.  I'll fix this locally as I suggested and get a fix merged ASAP.

Reply:
Should be good as of 0a789ea8a

Reply:
Thanks!